test.bats

```diff
@@ -67,6 +76,15 @@ teardown() {
     ls puck/flask
 }
 
+@test "py3.6 can package flask with slim & dockerizePip option" {
+    cd tests/base
+    npm i $(npm pack ../..)
+    ! uname -sm|grep Linux || groups|grep docker || id -u|egrep '^0$' || skip "can't dockerize on linux if not root & not in docker group"
+    sls --dockerizePip=true --slim=true package
+    unzip .serverless/sls-py-req-test.zip -d puck
```

Comment:
:raised_hands: Thanks for including tests too!!

Could you add this to also check that the unzipped services doesn't contain any pyc files?
```
test $(find . -name *.pyc | wc -l) -eq 0
```

Reply:
Sure, that's an awesome one, thanks, didn't know of this, will do now! :v:

Reply:
should I put `test $(find puck -name *.pyc | wc -l) -eq 0` maybe? so it runs inside of the `puck` folder?

Reply:
Yup. My mistake.

Reply:
@dee-me-tree-or-love, @dschep Wouldn't you want to dump the .py and not .pyc?